core/src/main/scala/dev/profunktor/auth/middleware.scala

```diff
@@ -14,6 +14,12 @@ object JwtAuthMiddleware {
   def apply[F[_]: MonadThrow, A](
       jwtAuth: JwtAuth,
       authenticate: JwtToken => JwtClaim => F[Option[A]]
+  ): AuthMiddleware[F, A] =
+    apply(Monad[F].pure(jwtAuth), authenticate)
```

Comment:
Nitpick: Could use `jwtAuth.pure[F]` here instead? You can remove the `Monad` import.

Reply:
done